### PR TITLE
chore: remove old/unused dev deps

### PR DESCRIPTION
**Pull request by holic**
Noticed these taking a while to install when wifi got shoddy.

### CODE REVIEW

**Comment by holic** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1068** <a href="https://app.graphite.dev/github/pr/latticexyz/mud/1068" target="_blank">Graphite</a> 
    * **PR #1067** <a href="https://app.graphite.dev/github/pr/latticexyz/mud/1067" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/latticexyz/mud/1067?utm_source=stack-comment).

File: packages/common/src/codegen/render-solidity/renderTypeHelpers.ts
Comment (alvrs) [inline packages/common/src/codegen/render-solidity/renderTypeHelpers.ts:1]: this change seems like it should go in #1068?

Comment (holic) [reply packages/common/src/codegen/render-solidity/renderTypeHelpers.ts:1]: yeah I dunno what happened, the other PR had this issue too

Comment (holic) [reply packages/common/src/codegen/render-solidity/renderTypeHelpers.ts:1]: image

image

I think git is accurate but github isn't?

Comment (holic) [reply packages/common/src/codegen/render-solidity/renderTypeHelpers.ts:1]: refreshing the base branch of the PR seemed to fix it

**Review by alvrs — APPROVED** [review]

**Pull request merged**